### PR TITLE
Tested multiple mmpks and made appropriate changes

### PR DESCRIPTION
**Pull request by marastoica**
@jNery please review. I tested the app with a couple of new mmpks and found a few issues that I fixed:
- Moved zoom scale for labels workaround into settings
- Added ability to use online basemaps 
- Changed how layer visibility is determined to account for more than 2 operational layers
- Replaced use of GeometryEngine.LabelPoint because it fails with true curves
- Set map background and removed grid
- Changed the way bottom contact card is generated to account for only one display field
- Replaced reference to nuget package v 100.0.0.1 with reference to v 100.0.0
- Changed floor picker selected cell color to blue

### CODE REVIEW

File: iOS/Controllers/MapViewController.cs [hunk 1]
```diff
@@ -567,23 +572,29 @@ private void MapView_GeoViewDoubleTapped(object sender, GeoViewInputEventArgs e)
                         var roomMarker = new PictureMarkerSymbol(new RuntimeImage(mapPin));
                         roomMarker.OffsetY = uiImagePin.Size.Height * 0.65;
 
-                        // Create graphic
-                        var mapPinGraphic = new Graphic(GeometryEngine.LabelPoint(idResults.GeoElements.First().Geometry as Polygon), roomMarker);
+                        // Create graphic
+                        var mapPinGraphic = new Graphic(idResults.GeoElements.FirstOrDefault().Geometry.Extent.GetCenter(), roomMarker);
```
Comment (jNery) [inline iOS/Controllers/MapViewController.cs]: `idResults` can be empty, right? If no feature was hit during identify or layer is not FeatureLayer?

File: iOS/Controllers/MapViewController.cs [hunk 2]
```diff
@@ -567,23 +572,29 @@ private void MapView_GeoViewDoubleTapped(object sender, GeoViewInputEventArgs e)
...
-                        graphicsOverlay.Graphics.Add(mapPinGraphic);
+                        graphicsOverlay.Graphics.Add(mapPinGraphic);
 
                         // Get room attribute from the settings. First attribute should be set as the searcheable one
-                        var roomAttribute = AppSettings.CurrentSettings.ContactCardDisplayFields[0];
-                        var employeeNameAttribute = AppSettings.CurrentSettings.ContactCardDisplayFields[1];
-                        var roomNumber = idResults.GeoElements.First().Attributes[roomAttribute];
-                        var employeeName = idResults.GeoElements.First().Attributes[employeeNameAttribute];
+                        var roomAttribute = AppSettings.CurrentSettings.ContactCardDisplayFields[0];
+                        var roomNumber = idResults.GeoElements.First().Attributes[roomAttribute];
```
Comment (jNery) [inline iOS/Controllers/MapViewController.cs]: If you intend to use idResults.GeoElements.FirstOrDefault() again, maybe put in a variable? Currently it's geometry is used for the pin graphic and it's attribute is retrieved.

File: iOS/Controllers/MapViewController.cs [hunk 3]
```diff
@@ -567,23 +572,29 @@ private void MapView_GeoViewDoubleTapped(object sender, GeoViewInputEventArgs e)
...
                         if (roomNumber != null)
-                        {
-                            var employeeNameLabel = employeeName ?? string.Empty;
+                        {
+                            var employeeNameLabel = string.Empty;
+                            if (AppSettings.CurrentSettings.ContactCardDisplayFields.Count > 1)
+                            {
+
+                                var employeeNameAttribute = AppSettings.CurrentSettings.ContactCardDisplayFields[1];
+                                var employeeName = idResults.GeoElements.First().Attributes[employeeNameAttribute];
```
Comment (jNery) [inline iOS/Controllers/MapViewController.cs]: The cached idResults.GeoElements.FirstOrDefault() can be used here if it were stored in a variable.

File: iOS/Controllers/MapViewController.cs (hunk 1 above)
Comment (marastoica) [inline iOS/Controllers/MapViewController.cs]: Correct. If it's empty, I catch the exception and hide the contact card.

**Pull request merged**